### DISCUSSION TITLE
Veryyyyyyyyy Slow transfer speed

### DISCUSSION THREAD

**Discussion by RoyalTurtleMan** [iterate-ch/cyberduck#16293] [Q&A]
![image](https://github.com/user-attachments/assets/64224c12-c13c-4e1f-9f8a-a2c43cfdf12f)
its gone from estimated to finish from 230hours to 310hours in 30 minutes and it keeps getting worse

**Comment by tatze96** [comment]
Please add the following information:
- Which operating system is in use?
- Which version of Cyberduck is in use?
- Which protocol is in use?
- Is this an upload or download process?
